src/vaults/mainnet.json

```diff
@@ -384,6 +384,31 @@
       "beraRewardsVault": "0x4eA84882228a5C881675151E951235e45256a484",
       "slug": "wasabi-swbera",
       "stakeTokenAddress": "0xc95ab9eff8fb48760703c74416764b8f898afa1b"
+    },
+    {
+      "beraRewardsVault": "0x8412561FADDD6C85F5a17e07E0F29428B813E69B",
+      "slug": "beradrome-henlocker100m",
+      "stakeTokenAddress": "0x8412561FADDD6C85F5a17e07E0F29428B813E69B"
```

Comment:
also this is the same address as the rewards vault

Reply:
thanks!
however i still need all the data for the underlying tokens including images